README.md

```diff
@@ -147,14 +145,14 @@ jobs:
 
 ## Compressing images on a schedule
 
-It is also possible to run image-actions on a reoccurring schedule. By using the `compressOnly` option,  in conjunction with [@peter-evans's](/peter-evans) [`create-pull-request`](https://github.com/peter-evans/create-pull-request) action, a new Pull Request will be raised if there are optimised images in a repository.  
+It is also possible to run image-actions on a reoccurring schedule. By using the `compressOnly` option, in conjunction with [@peter-evans's](/peter-evans) [`create-pull-request`](https://github.com/peter-evans/create-pull-request) action, a new Pull Request will be raised if there are optimised images in a repository.

```

Comment:
```suggestion
It is also possible to run image-actions on a recurring schedule. By using the `compressOnly` option, in conjunction with [@peter-evans's](/peter-evans) [`create-pull-request`](https://github.com/peter-evans/create-pull-request) action, a new Pull Request will be raised if there are optimised images in a repository.
```

> Something that is recurring happens over and over again, possibly at regular intervals. In contrast, something that is reoccurring is simply happening again but not always repeatedly.
[source](https://www.grammarly.com/blog/recur-reoccur/)

Reply:
Are you saying I shouldn't invent words? 🤪